src/server/pmix_server_get.c

```diff
@@ -199,7 +199,14 @@ pmix_status_t pmix_server_get(pmix_buffer_t *buf,
          * give the host server a chance to tell us about it */
         rc = create_local_tracker(nspace, rank, info, ninfo,
                                   cbfunc, cbdata, &lcd);
-        return rc;
+/*
+ * Its possible there are no local processes on this host, so lets ask 
+ * for this explicitly.   
+ */
+        if( NULL != pmix_host_server.direct_modex ){
+                pmix_host_server.direct_modex(&lcd->proc, info, ninfo, dmdx_cbfunc, lcd);
```

Comment:
Maybe? I don't know - my point was only that we should check to ensure that if we get the desired info via registration prior to receiving the direct modex response, then we need to ensure we don't pass the info down to the requesting client twice